### PR TITLE
Added asset version information

### PR DESCRIPTION
**Pull request by core23**
I am targeting this branch, because this is pedantic.

## Changelog

```markdown
### Added
- Added `package.json` file for asset version information
```

## Subject

This is the first step to remove external assets from this bundle.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@sonata-project/formatter-bundle",
+  "version": "0.1.0",
+  "homepage": "https://sonata-project.org/bundles/formatter",
+  "description": "Sonata Formatter Bundle assets",
+  "dependencies": {
+    "ckeditor": "^4.3",
+    "font-awesome": "^4.7",
+    "ionicons": "^2.0",
+    "markitup": "^3.0.0-0"
```
Comment (kunicmarko20) [inline package.json]: we are using 1.x version of this

Comment (core23) [reply package.json]: There is no 1.x version in the npm:
https://www.npmjs.com/package/markitup

Comment (kunicmarko20) [reply package.json]: what now?

**Comment by core23** [comment]
We won't need this after merging #389

**Pull request closed without merging**